daft/dataframe/dataframe.py

```diff
@@ -549,62 +549,109 @@ def write_iceberg(self, table: "pyiceberg.table.Table", mode: str = "append") ->
         # This is due to the fact that the logical plan of the write_iceberg returns datafiles but we want to return the above data
         return with_operations
 
+    @DataframePublicAPI
     def write_delta(
```

Comment:
I think this should be consistent with `read_delta_lake` so lets rename to `write_delta_lake`?